php/ui/component/class-submit.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * Submit UI Component.
+ *
+ * @package Cloudinary
+ */
+
+namespace Cloudinary\UI\Component;
+
+use Cloudinary\UI;
+
+/**
+ * Class Component
+ *
+ * @package Cloudinary\UI
+ */
+class Submit extends UI\Component {
```

Comment:
@DavidCramer 

Can we declare this in the `use` so it remains only the `Component`